spec/library_spec.rb

```diff
@@ -0,0 +1,35 @@
+require './lib/library.rb'
+
+describe Library do
+books = [{ title: 'Alfons leker', author: 'A. Andersson' },{ title: 'Bible', author: 'TGAOTU' } ]
+  subject { described_class.new(books: books)}
+  #let(:user) { double('User') }
+
+
+ it 'has a collection of books' do
+    expect(subject.books).to be_kind_of Array
+  end
```

Comment:
Lint/BlockAlignment: `end` at 11, 2 is not aligned with `it 'has a collection of books' do` at 9, 1.